modules/installation-vsphere-machines.adoc

```diff
@@ -42,8 +42,8 @@ If you plan to add more compute machines to your cluster after you finish instal
 ====
 
 ifndef::openshift-origin[]
-. Obtain the {op-system} OVA image from the link:https://access.redhat.com/downloads/content/290[Product Downloads] page on the Red Hat Customer Portal or the link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.7/[{op-system} image mirror] page.
-+
+. Obtain the {op-system} OVA image version that matches your {product-title} version if it is available. Images are available from the link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/[{op-system} image mirror] page, under `your-openshift-container-platform-version/latest/`
```

Comment:
Suggested edit to match the file name: 

s/ `your-openshift-container-platform-version/latest/` / `/pub/openshift-v4/dependencies/<op-system>/latest/`.